public/route/style.css

```diff
@@ -2,55 +2,100 @@ html,
 body {
   padding: 0;
   margin: 0;
-  color: #323c4d;
-  font-family: Inter, sans-serif;
+  color: #1b2023;
+  font-family: 'Inter', 'Roboto', sans-serif;
   font-size: 16px;
   font-weight: normal;
 }
 
 * {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
 }
 
-span {
-  font-weight: 600;
-}
-
-ul,
-ol {
-  padding: 0;
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
   margin: 0;
-  list-style: none;
+  padding: 0;
 }
 
 h1 {
-  margin: 0;
-  font-size: 16px;
-  font-weight: 200;
-  line-height: 24px;
+  font-weight: bold;
+  font-size: 2rem;
+  line-height: 2.5rem;
 }
 
-h2 {
-  padding: 8px 16px 0;
-  margin-bottom: 10px;
-  font-size: 18px;
+h3 {
   font-weight: 600;
-  line-height: 24px;
+  font-size: 1.25rem;
```

Comment:
We are using px everywhere else. Let's stick to that.

Reply:
Hm, I switched like everything to REM / EM. We doing the same on the website actually. The reason I did this is whenever we adjust our base-font or spacing will be relative without having to re-do the whole UI again.

Was about to do something similar to the other examples. Planning on moving the colors to the `:root` as well. If we really prefer `px` I'll revert the changes.

Reply:
I think rem is okay too. But then indeed we should be using it everywhere.